dts/riscv/ite/it82xx2.dtsi

```diff
@@ -390,6 +390,8 @@
 				      NO_FUNC 0
 				      NO_FUNC 0>;
 			interrupt-parent = <&intc>;
+			extended-set = <BIT(7)   BIT(7)   BIT(7)   BIT(7)
```

Comment:
I'm not sure it makes sense to specify an 8-bit bit mask for every pin on the keyboard GPIOs controllers.   What about about just adding a bool property that identifies this controller as a keyboard type so that the driver can make the correct register updates?

Reply:
Use Boolean properties instead.